### PR TITLE
Radioactive decay

### PR DESCRIPTION
**Pull request by TheDoctorRAB**
I created a new branch for this node now. I've transferred all the content and exercises I have on radioactive decay.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +0,0 @@
-_site/
```
Comment (gonuke) [inline .gitignore:1]: Should not remove this file.  Rather - it should simply not be included in the pull request.

File: radioactive-decay/info.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+uid: radioactive_decay 
+title: radioactive decay 
+prerequisites:
+  - undergraduate statistics and probability
+learning_objectives:
+  - basic understanding of radioactivity
```
Comment (gonuke) [inline radioactive-decay/info.yml]: (continuing to use this as a way to discuss some of the issues about scope, etc, ...)
This continues to seem like a broad learning objective.

Looking at the content, it seems that the focus is on the time behavior of radioactive decay, and none of the nuclear physics.  I think the scope is probably what I would expect, but the statement of learning objective does not match that scope.  Might also need some additional pre-requisites.

Interesting question on pre-requisites: Some instructors may choose different orders (deductive vs inductive style) and thus indicate pre-requesites in a completely different order.  For example, related to this node - some instructors might choose to discuss some aspects of decay (types of decay) before this node and others might do it after this node.

File: radioactive-decay/info.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+references:
+  - lamarsh.bib
+abet_outcomes: n/a
+assessments: 
+  - radioactive.decay.md
```
Comment (gonuke) [inline radioactive-decay/info.yml]: You have a few assessments, and none of them are this one...

**Review by gonuke — COMMENTED** [review]
More practice at reviewing NECX node PRs

File: .gitignore (hunk 1 above)
Comment (TheDoctorRAB) [reply .gitignore:1]: I can't figure out how to do it, and it was getting annoying.

File: radioactive-decay/info.yml (hunk 2 above)
Comment (TheDoctorRAB) [reply radioactive-decay/info.yml]: It's ok if we combine this content with a more in depth node on decay too. I was just going to take all the content I have and put it up for everyone to use. I do link to other material that covers decay in more detail in the OER.

File: radioactive-decay/info.yml (hunk 3 above)
Comment (TheDoctorRAB) [reply radioactive-decay/info.yml]: I just forgot to include that. I'll add the other files.
